util/telemetry/src/lib.rs

```diff
@@ -117,7 +116,9 @@ cfg_if::cfg_if! {
 
             opentelemetry_jaeger::new_pipeline()
                 .with_service_name(service_name)
-                .with_trace_config(Config::default().with_resource(Resource::new(tags)))
+                .with_trace_config(
+                    sdk::trace::Config::default()
+                        .with_resource(sdk::Resource::new(tags)))
```

Comment:
Another way to fix this is to move the use statements into the cfg if block with the other use statements for this block